ChallengesWithTestsMarkVII/ChallengesSet05.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Text;
+
+namespace ChallengesWithTestsMarkVII
+{
+    public class ChallengesSet05
+    {
+        public int GetNextNumberDivisibleByN(int startNumber, int n)
+        {
+            if (startNumber == 0)
+            {
+                return n;
+            }
+
+            List<int> firstlist = new List<int>();
+            int maxLoop = startNumber * n;
+            int[] numbersDivisblebyN = new int[0];
+
+            for (int i = 0; i < maxLoop; i++)
+            {
+                if (i > startNumber && i % n == 0)
+                {
+                    firstlist.Add(i);
+                }
+
+            }
+
+            firstlist.ToArray();
+            return firstlist[0];
+
+        }
+
+        public void ChangeNamesOfBusinessesWithNoRevenueTo_CLOSED(Business[] businesses)
+        {
+            for (int i = 0; i < businesses.Length; i++)             {                 if (businesses[i].TotalRevenue == 0 || businesses[i].TotalRevenue == null)                 {                     businesses[i].Name = "CLOSED";                 }             }         }
```

Comment:
`TotalRevenue` is a double and will thus never be null, so that part of this if-statement should be removed